diff --git a/api.js b/api.js
index 624e834abe70dc59e75344927922ab585fcb0439..5281829b3c75f37095dba7085eabf875de0c03fc 100644
--- a/api.js
+++ b/api.js
@@ -42202,9 +42202,9 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
diff --git a/api.mjs b/api.mjs
index df1c48e05be38829e97f175445c374ab28334dc4..e9240d7042ef790323a0940909b69ff1797bd1c2 100644
--- a/api.mjs
+++ b/api.mjs
@@ -42207,9 +42207,9 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
diff --git a/bin.cjs b/bin.cjs
index 27f1406f7e68279ade9e4c95ad023ac44ab6a714..d004c8bf6aa8efe2ebe0db9825c3650442f22ed8 100755
--- a/bin.cjs
+++ b/bin.cjs
@@ -18394,9 +18394,9 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
